src/main/java/digital/slovensko/autogram/core/SigningParameters.java

```diff
@@ -188,13 +190,13 @@ public static SigningParameters buildFromRequest(SignatureLevel level, ASiCConta
             String containerXmlns, SignaturePackaging packaging, DigestAlgorithm digestAlgorithm,
             Boolean en319132, String infoCanonicalization, String propertiesCanonicalization,
             String keyInfoCanonicalization, String schema, String transformation, String identifier,
-            boolean checkPDFACompliance, int preferredPreviewWidth, boolean autoLoadEform, String xsdIdentifier,
-            String xsltIdentifier, String xsltLanguage, String xsltType, String xsltTarget, DSSDocument document,
-            TSPSource tspSource) throws AutogramException {
+            boolean checkPDFACompliance, int preferredPreviewWidth, boolean autoLoadEform, boolean embedUsedSchemas,
+            String xsdIdentifier, String xsltIdentifier, String xsltLanguage, String xsltType, String xsltTarget,
+            DSSDocument document, TSPSource tspSource) throws AutogramException {
 
         return buildParameters(level, container, containerXmlns, packaging, digestAlgorithm, en319132,
```

Comment:
Toto uz smrdi nejakym builder patternom, tie positional optional boolean parameters su dost peklicko.

Reply:
Súhlasím. Ale podľa mňa problém na iný deň.